src/main/java/com/ning/http/util/AuthenticatorUtils.java

```diff
@@ -152,6 +142,44 @@ public static String perRequestProxyAuthorizationHeader(Request request,
 
         return proxyAuthorization;
     }
+
+    private static String getNtlmProxyAuthorization(Request request, ProxyServer proxyServer, boolean connect) {
```

Comment:
Put a comment indicating that one of this methods along with the sysprop has to be removed in the next cycle and create a task for it.

Reply:
Done.